src/Pim/Bundle/CatalogBundle/EventSubscriber/IndexProductsSubscriber.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Pim\Bundle\CatalogBundle\EventSubscriber;
+
+use Akeneo\Bundle\ElasticsearchBundle\Client;
+use Akeneo\Component\StorageUtils\StorageEvents;
+use Pim\Component\Catalog\Model\ProductInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\EventDispatcher\GenericEvent;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+/**
+ * Index products in the search engine.
+ *
+ * This is not done directly in the product saver as it's only a technical
+ * problem. The product saver only handles business stuff.
+ *
+ * @author    Julien Janvier <j.janvier@gmail.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class IndexProductsSubscriber implements EventSubscriberInterface
+{
+    /** @var NormalizerInterface */
+    protected $normalizer;
+
+    /** @var Client */
+    protected $indexer;
+
+    /** @var string */
+    protected $indexType;
+
+    /**
+     * @param NormalizerInterface $normalizer
+     * @param Client              $indexer
+     * @param string              $indexType
+     */
+    public function __construct(NormalizerInterface $normalizer, Client $indexer, $indexType)
+    {
+        $this->normalizer = $normalizer;
+        $this->indexer = $indexer;
+        $this->indexType = $indexType;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public static function getSubscribedEvents()
+    {
+        return [
+            StorageEvents::POST_SAVE => 'indexProduct',
```

Comment:
all those events (events by entity) have been dropped in 1.5 (or 1.4?) by clement
this PR is not about changing the events we trigger

here I use what is at my disposal to index products